lawluigi_configs/KingMaker_law.cfg

```diff
@@ -17,12 +17,14 @@ max_reschedules: 3
 [job]
 job_file_dir = $ANALYSIS_DATA_PATH/jobs
 job_file_dir_cleanup: False
+job_file_dir_mkdtemp: True
 
 [target]
 default_wlcg_fs = wlcg_fs
 
 [wlcg_fs]
-base: srm://cmssrm-kit.gridka.de:8443/srm/managerv2?SFN=/pnfs/gridka.de/cms/disk-only/store/user/${USER}/CROWN_samples
+base: root://cmsxrootd-kit.gridka.de//store/user/${USER}/CROWN_samples
 use_cache: True
 cache_root: /tmp/$USER/
-cache_max_size: 20000
\ No newline at end of file
+cache_max_size: 20000
+gfal_atomic_contexts: True
```

Comment:
it opens a new context (new connection) for every gfal interaction with the storage, closing the connection after each use. It is a recommended setting when running larger tasks to avoid too many open connections and therefore load on the storage